.gitlab-ci.yml

```diff
@@ -0,0 +1,62 @@
+include: [component: $CI_SERVER_FQDN/lc-components/id_tokens/id_tokens-component@main]
+      
```

Comment:
Perhaps add a comment that this is written to work with the CI infrastructure at LLNL and likely will not work at other sites that use gitlab.

Reply:
done.